src/index.ts

```diff
@@ -19,8 +19,8 @@ export async function generateEosPaths() {
     await buildPathsFile();
 }
 
-export async function generatePath(sourceToken: Token, targetToken: Token) {
-    return await generatePathByBlockchainIds(sourceToken, targetToken);
+export async function generatePath(sourceToken: Token, targetToken: Token, amount: string = '1', getBestPath: (paths: string[][], rates: string[]) => string[] = getEthCheapestPath) {
```

Comment:
I don't think we should expose the ability to pass in a function here, it's too much.
If users want to do custom path generation, they can always get all the paths and do whatever they want.

Reply:
It's either a function, or some "enum" representing the function.

Reply:
so let's do enum, that can work